server/main.go

```diff
@@ -55,7 +59,10 @@ func getWebHandler(activeFileManager *ActiveFileManager, fileStore FileStore) ht
 			} else if method == "PUT" {
 				// uploading a file
 				handlePutFile(res, req, path[0], activeFileManager)
+			} else if method == "OPTIONS" {
+				return
 			} else {
+				res.Header().Set("Allow", "GET, PUT, OPTIONS")
```

Comment:
I haven't seen that it's mandatory in the spec. In my testing, it wasn't needed.